Misc/NEWS.d/next/Library/2024-10-24-15-21-03.gh-issue-125889.P9MXpN.rst

```diff
@@ -0,0 +1,2 @@
+Modify the check of the ``hi`` argument in ``internal_bisect_left`` and ``internal_bisect_right``
+so that when ``hi`` is negative, it's calculated based on the list length.
```

Comment:
This NEWS entry is not useful for end-users. The `internal_bisect_left/right` functions are internal. What should be said is whether negative values for `hi` are now allowed and behave as one would expect (i.e., negative hi = count from the end).